bliss/conf/base_config.yaml

```diff
@@ -27,6 +27,29 @@ simulator:
     num_workers: 32
     valid_n_batches: 10  # 256
     fix_validation_set: true
+    prior:
+        _target_: bliss.simulator.prior.CatalogPrior
+        reference_band: ${simulator.survey.reference_band}
+        max_bands: ${simulator.survey.max_bands}
+        color_model_dir: ${simulator.survey.dir_path}/color_models
```

Comment:
as discussed, rather than a `color_model_dir`, we'll specify file paths to a star color model and a galaxy color model